payments-ui-core/src/main/java/com/stripe/android/ui/core/IsStripeCardScanAvailable.kt

```diff
@@ -0,0 +1,16 @@
+package com.stripe.android.ui.core
```

Comment:
I think most of this could live in PaymentSheet, we can always move it to core if other modules want to share it. Thoughts? Maybe link might want it in the future.

Reply:
If I move this and the proxy to PaymentSheet, the CardScanActivity in ui.core would need a dependency on paymentsheet to run